CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+The following is a set of guidelines for contributing to KNPPaginatorBundle, which is hosted by the [KNP Labs Organization][KNPLink] on GitHub. These are
+mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document opening a pull request. :shipit:
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the following [Code of Conduct][CodeofConduct]. By participating, you are expected to uphold this code.
+
+## Reporting a bug
+
+#### Before submitting a bug
+- Verify that you are using the latest KNPPaginatorBundle version;
+- Double-check the [documentation][ReadMe] to see if you're not misusing the library;
+- Check if the issue has already been reported. If it has and the issue is still open, add a comment to the existing issue instead of opening a new one.
+
+#### How to submit a (good) bug report
+To report a KNPPaginatorBundle bug please open a [GitHub issue][GitHub-issue] following the rules below.
+
+- Use a clear and descriptive title for the issue to identify the problem;
+- Describe the steps needed to reproduce the bug including a code example when possible;
+- Give as much detail as possible about your environment (OS, PHP version, KNPPaginatorBundle configuration, ...);
+
+## Suggesting enhancements
+
+To suggest KNPPaginatorBundle enhancements please open a [GitHub issue][GitHub-issue] following the rules below.
+
+- Use a clear and descriptive title for the issue to identify the problem;
+- Provide a step-by-step description of the suggested enhancement in as many details as possible;
+- Explain why this enhancement would be useful with one or more use cases;
+
+## Contributing to the code
+
+A pull request, is the best way to provide a bug fix or to propose enhancements to KNPPaginatorBundle.
+
+When submitting a pull request please be sure to follow the same rules described above in [Reporting a bug](#reporting-a-bug) and [Suggesting enhancements](#suggesting-enhancements)
+sections depending on the nature of your change.
+
+> Before starting to work on a large change please open an issue to ask the maintainers if they are fine with it (no one likes to work for nothing!).
+
+1. Fork the repository
+2. Once the repository has been forked clone it locally
+```
+git clone git@github.com:USERNAME/KnpPaginatorBundle.git
+```
+3. Create a new branch
+```
+git checkout -b BRANCH_NAME master
+```
+4. Code!!!
+5. Add/Update tests (if needed)
+6. Update documentation (if needed)
+7. Run the tests and make sure that they are passing
+```
+vendor/bin/phpunit
+```
+8. Squash your commits
+9. Rebase your branch on master and fix merge conflicts
+10. Open the pull request
+
+[GitHub-issue]: https://github.com/KnpLabs/KnpPaginatorBundle/issues
+[ReadMe]: https://github.com/KnpLabs/KnpPaginatorBundle/blob/master/README.md
+[KNPLink]: https://github.com/KnpLabs
+[CodeofConduct]: https://github.com/KnpLabs/KnpPaginatorBundle/blob/normalize-code-of-conduct/CODE_OF_CONDUCT.md
```

Comment:
I guess this is not the final link (your branch will likely be deleted after merging)

Reply:
oups, sorry, changed that !